glossary.asciidoc

```diff
@@ -341,18 +336,18 @@ P2WPKH::
     The signature of a P2WPKH (Pay-to-Witness-Public-Key-Hash) contains the same information as a P2PKH spending, but is located in the witness field instead of the scriptSig field. The scriptPubKey is also modified.
 
 P2WSH::
-    The difference between P2SH and P2WSH (Pay-to-Witness-Script-Hash) is about the cryptographic proof location change from the scriptSig field to the witness field and the scriptPubKey that is also modified.
+    The difference between P2SH and P2WSH (Pay-to-Witness-Script-Hash) is the location change of the cryptographic proof from the scriptSig field to the witness field and the modification of the scriptPubKey.
 
 payment::
-    A payment occurs if we transfer bitcoin within the Lightning Network.
+    A payment occurs if bitcoin is stransferred within the Lightning Network.
```

Comment:
There is a typo here

Reply:
Yes, sorry.  Glossaries should be neutral.  I just think "we" should be removed.

Reply:
Sure, I agree with the correction, I just wanted to signal the typo before it gets merged :)

Reply:
By the way, I'm not sure what I should do about typos like this.  Is there a way I can correct it in the pull request, or do I just leave it for someone else to sort out?  (I'm a GitHub newbie!)  Guidance welcome.

Reply:
To be honest, I'm not a pro either :)
What I think you should do is: 
* do the changes that you want to make on your repository
* commit the changes as a new commit
* push the commit
This should appear as a new commit in this pull request, or at least I think so.